src/layouts/MiddleWrapper/MiddleWrapper.jsx

```diff
@@ -0,0 +1,16 @@
+import ContentWrapper from '../ContentWrapper/ContentWrapper';
+import SidebarWrapper from '../SIdebarWrapper/SIdebarWrapper';
```

Comment:
I --> i in '../SIdebarWrapper/SIdebarWrapper'

Reply:
fixed